src/editors/object.js

```diff
@@ -291,7 +291,7 @@ JSONEditor.defaults.editors.object = JSONEditor.AbstractEditor.extend({
     // If the object should be rendered as a div
     else {
       this.header = document.createElement('span');
-      this.header.textContent = this.getTitle();
+      this.header.textContent = this.schema.plural_title || this.schema.title || this.getTitle();
```

Comment:
If `getTitle` in `editor.js` now returns `plural_title` and `title` preferentially over the key, is this change still necessary?